### ISSUE TITLE
get_as_df not retrieving all columns in table

### ISSUE THREAD

**Issue by ashwinramdas12** [nithinmurali/pygsheets#434]
I have been calling get_as_df() for a while with no issues, but then when i installed the stage version (https://github.com/nithinmurali/pygsheets/archive/staging.zip) I get this error. I have a table that stretches from column A to J, but when i called get_as_df, it only grabs til column I. I've tried specifiying an end that is beyond J, and column J has values in it. Did something change in the staging version to cause this?

**Comment by ashwinramdas12** [comment]
This has happened for multiple worksheets now. Why is this happening all of a sudden?

**Comment by ashwinramdas12** [comment]
It's just arbitrarily deciding how many columns to pull in instead of just pulling in all the columns. This is breaking my code badly. Please help

**Comment by fstinner** [comment]
I had the same problem. The reason is the parameter `numerize`. It must be put on `False`.

```
self.ws.get_as_df(has_header=True, include_tailing_empty=True, numerize=False)
```

**Comment by nithinmurali** [comment]
fixed in #436
